packages/dai-plugin-migrations/test/helpers/mocks.js

```diff
@@ -0,0 +1,54 @@
+import BigNumber from 'bignumber.js';
+import { RAY } from '../../src/constants';
+import { bytesToString } from './';
+
+export const globalSettlement = {
+  beforeCage: () => ({
+    live: async () => true,
+    tags: async () => new BigNumber(0),
+    fix: async () => new BigNumber(0)
+  }),
+  afterCage: () => ({
+    live: async () => false,
+    tags: async () => new BigNumber(0),
+    fix: async () => new BigNumber(0)
+  }),
+  afterCageCollateral: tags => ({
+    live: async () => false,
+    tags: async ilk => {
+      const ilkAsString = bytesToString(ilk);
+      return tags[ilkAsString]
+        ? RAY.times(new BigNumber(1).div(tags[ilkAsString]))
+        : new BigNumber(0);
+    },
+    fix: async () => new BigNumber(0)
+  }),
+  afterFlow: fixes => ({
+    live: async () => false,
+    fix: async ilk => {
+      const ilkAsString = bytesToString(ilk);
+      return fixes[ilkAsString]
+        ? RAY.times(fixes[ilkAsString])
+        : new BigNumber(0);
+    }
+  })
+};
```

Comment:
It'd be nice to not rely on mocks for testing global settlement. I'm talking to Gonzalo about this. Deployment code suggests that leaving an authed user around to kick off settlement would be super not straight-forward, but maybe he can think of a better way?